huaweicloud/services/dms/resource_huaweicloud_dms_rocketmq_instance.go

```diff
@@ -580,7 +582,7 @@ func resourceDmsRocketMQInstanceDelete(ctx context.Context, d *schema.ResourceDa
 	return nil
 }
 
-func DmsRocketmqInstanceStateRefreshFunc(client *golangsdk.ServiceClient, instanceID string) resource.StateRefreshFunc {
+func RocketmqInstanceStateRefreshFunc(client *golangsdk.ServiceClient, instanceID string) resource.StateRefreshFunc {
```

Comment:
```suggestion
func rocketMQInstanceStateRefreshFunc(client *golangsdk.ServiceClient, instanceID string) resource.StateRefreshFunc {
```
It seems it should be a private function

Reply:
done